src/components/ProjectWork/ConfirmationCheck.js

```diff
@@ -1,198 +1,149 @@
 import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
 
 const ConfirmationCheck = ({ hasConfirmed, setHasConfirmed }) => {
-  const [confirmationChecks, setConfirmationChecks] = useState([false, false, false]);
+  const [confirmationChecks, setConfirmationChecks] = useState([
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false,
+    false
+  ]);
   const [allChecked, setAllChecked] = useState(false);
 
   useEffect(() => {
-    if (confirmationChecks.every((check) => check)) {
-      setAllChecked(true);
-    }
+    setAllChecked(confirmationChecks.every((check) => check));
   }, [confirmationChecks]);
 
+  const updateCheckbox = (id) => {
+    setConfirmationChecks(confirmationChecks.map((check, index) => (index === id ? !check : check)));
+  };
+
   return (
     <div className="confirmation-checks-modal">
-      <h4>Before you take on this project</h4>
-      <p>
+      <h3>Before you take on this project</h3>
+      <p className="indent-text">
```

Comment:
can we update all the classnames in this file to use the same prefix (ie `confirmation-`) so that there definitely won't be any overlaps

Reply:
oh i see now that indent-text is like a global thing but the comment stands for the rest of the classes in this file